iset.mm

```diff
@@ -100691,15 +100691,15 @@ seq K ( .+ , G ) ) $=
         UPYOWGXBUVMYNWHWIWJABCDXFYJUUTFELWKWLVHUMWNVKWOWPWQWOWRWSWT $.
     $}
 
-    iseq1p.4 $e |- ( ph -> M e. ZZ ) $.
-    iseq1p.5 $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( F ` x ) e. S ) $.
+    seq3-1p.4 $e |- ( ph -> M e. ZZ ) $.
+    seq3-1p.5 $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( F ` x ) e. S ) $.
     $( Removing the first term from a sequence.  (Contributed by Jim Kingdon,
        16-Aug-2021.) $)
-    iseq1p $p |- ( ph -> ( seq M ( .+ , F , S ) ` N ) =
-        ( ( F ` M ) .+ ( seq ( M + 1 ) ( .+ , F , S ) ` N ) ) ) $=
-      ( cseq4 cfv c1 caddc co wcel cz cuz uzid syl iseqsplit iseq1 oveq1d eqtrd
-      ) AIEFGHOZPHUIPZIEFGHQRSOPZESHGPZUKESABCDEFGHHIJKLAHUATHHUBPTMHUCUDNUEAUJ
-      ULUKEABCEFGHMNJUFUGUH $.
+    seq3-1p $p |- ( ph -> ( seq M ( .+ , F ) ` N ) =
```

Comment:
Is that a temporary label ? Not sure about our conventions regarding dashes in labels, except `ax-` and `df-`. As for me, I would prefer to avoid them.

Reply:
There's a decent amount of precedent for hyphens especially in many cases between two digits. This is partly written down at https://us.metamath.org/mpeuni/conventions-labels.html but I'm basing it at least as much on what I see in set.mm and iset.mm theorems that I didn't name.

As for whether the `seq3` and `sum3` convention should persist after we are done converting `seq` syntax, I'm not really sure. If someone has a better idea I'd probably be open to it, but using `iseq` and `isum` (prefixed `i` being our fallback for when set.mm and iset.mm differ) is kinda not great when `isum` is used in set.mm for infinite sum (as opposed to `fsum` for finite sum). I'm not really eager to double down on the `iisum` names I have used occasionally for "intuitionistic infinite sum".

Reply:
Ok.  I guess `seq3` will become `seq` when the transition is over ?

Reply:
In some cases, like `seqf` or `seqeq1`, it already is `seq`.

But the hard part is something like https://us.metamath.org/mpeuni/seq1.html . Assuming we follow "Theorems which are different should be named differently" from https://us.metamath.org/ileuni/conventions.html it can't be `seq1` . Right now it is https://us.metamath.org/ileuni/seq3-1.html which is perhaps an odd convention but is at least documented in the table at the end of https://us.metamath.org/ileuni/conventions.html . Maybe it is OK unless we think of something better?

Reply:
I thought you were adding an `i` in these cases, like `iseq1` ?

Reply:
> I thought you were adding an `i` in these cases, like `iseq1` ?

Perhaps but that's what I was discussing when I said:

> using iseq and isum (prefixed i being our fallback for when set.mm and iset.mm differ) is kinda not great when isum is used in set.mm for infinite sum (as opposed to fsum for finite sum). I'm not really eager to double down on the iisum names I have used occasionally for "intuitionistic infinite sum".